contracts-periphery/src/UmbraBatchSend.sol

```diff
@@ -1,11 +1,13 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.13;
 
-import "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
-import "openzeppelin-contracts/contracts/token/ERC20/utils/SafeERC20.sol";
+import "../lib/openzeppelin-contracts/contracts/access/Ownable.sol";
+import "../lib/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+import "../lib/openzeppelin-contracts/contracts/token/ERC20/utils/SafeERC20.sol";
```

Comment:
Why the change to relative paths?

Reply:
Was getting
```
Compiler run failed
error[6275]: ParserError: Source "openzeppelin-contracts/contracts/access/Ownable.sol" not found: File not found. Searched the following locations: ".../umbra-protocol/contracts-periphery".
 --> src/UmbraBatchSend.sol:4:1:
  |
4 | import "openzeppelin-contracts/contracts/access/Ownable.sol";
```

Reply:
Huh weird, this didn't used to happen though right? Either way, IMO it's preferable to solve this with remappings since relative paths aren't as nice to work with (things break if you move around files, less clear where the file comes from)

```toml
remappings = [
  "openzeppelin-contracts/=lib/openzeppelin-contracts/contracts"
]
```

```solidity
import "openzeppelin-contracts/access/Ownable.sol";
```

Reply:
Reminder that you only need to define it in the default profile and the CI profile will inherit that, so we currently have:

```toml
[profile.default]
verbosity = 3
fs_permissions = [{ access = "read", path = "./"}]
remappings = ['openzeppelin-contracts/=lib/openzeppelin-contracts/contracts']
[profile.ci]
fuzz_runs = 10000
fs_permissions = [{ access = "read", path = "./"}]
remappings = ["openzeppelin-contracts/=lib/openzeppelin-contracts/contracts"]
# See more config options https://github.com/gakonst/foundry/tree/master/config
```

But we only need:
```toml
[profile.default]
verbosity = 3
fs_permissions = [{ access = "read", path = "./"}]
remappings = ['openzeppelin-contracts/=lib/openzeppelin-contracts/contracts']

[profile.ci]
fuzz_runs = 10000
```

Reply:
gotcha, removed duplicates https://github.com/ScopeLift/umbra-protocol/pull/351/commits/88f80ca520f5b317c3fb77e0f0bcf0768b7d9a0e